wgsl/index.bs

```diff
@@ -4081,6 +4081,72 @@ TODO: user data inputs of fragment stage must be subset of user data outputs of
 
 ### Input-output matching rules TODO ### {#input-output-matching}
 
+# Language extensions # {#language-extensions}
+
+The [SHORTNAME] language is expected to evolve over time.
+
+An <dfn noexport>extension</dfn> is a named grouping for a coherent
+set of modifications to a particular version of the [SHORTNAME] specification, consisting of any combination of:
+* Addition of new concepts and behaviours via new syntax, including:
+    * declarations, statements, attributes, and built-in functions.
+* Removal of restrictions in the current specification.
+* Syntax for reducing the set of permissible behaviours.
+* A description of how the feature interacts with the existing specification, and optionally with other extensions.
+
+For example, extensions could be used to:
+* Add numeric scalar types, such as 16-bit integers.
+* Add syntax to constrain floating point rounding mode.
+* Add new kinds of statements.
+* Add new built-in functions.
+* Add constraints on how shader invocations execute.
+* Add new shader stages.
+
+An <dfn noexport>enable declaration</dfn> indicates that a particular named
```

Comment:
This sounds like it's mixing the concept of a "feature" versus "extension". Is this intended? Do you see WGSL extensions defining multiple features that need to be separately enabled?

Reply:
Whoops. I should have written "extension".

Or

>   ... indicates that a particular extension is in effect in the source text that follows.